### PR TITLE
Prepare 1.4.6 release

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

- Bump appwrite version to 1.4.6
- Add 1.4.6 changes to CHANGES.md
- Fix get database queue endpoint params
- Update specs for 1.4.6
- Generate 1.4.x docs examples

## Test Plan

Tested by creating a local appwrite/appwrite:1.4.6 image and installed it:

image

image

## Related PRs and Issues

None

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Comment by stnguyen90** [comment]
I do notice this on startup, though:

image

**Comment by stnguyen90** [comment]
Database deletes are now handled by database worker:

image

File: app/init.php [hunk 1]
```diff
@@ -110,7 +110,7 @@
 const APP_USER_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_CACHE_UPDATE = 24 * 60 * 60; // 24 hours
 const APP_CACHE_BUSTER = 513;
```
Comment (christyjacob4) [inline app/init.php:112]:
```suggestion
const APP_CACHE_BUSTER = 514;
```

File: docs/examples/1.4.x/console-cli/examples/health/get-queue-databases.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+appwrite health getQueueDatabases \
```
Comment (christyjacob4) [inline docs/examples/1.4.x/console-cli/examples/health/get-queue-databases.md:1]: I think we've to regenerate the example after we moved the param to the query?

**Review by abnegate — APPROVED** [review]

**Pull request merged**